pkg/kubelet/apis/well_known_labels.go

```diff
@@ -19,63 +19,41 @@ package apis
 import (
 	"strings"
 
+	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 const (
-	LabelHostname           = "kubernetes.io/hostname"
-	LabelZoneFailureDomain  = "failure-domain.beta.kubernetes.io/zone"
-	LabelMultiZoneDelimiter = "__"
-	LabelZoneRegion         = "failure-domain.beta.kubernetes.io/region"
-
-	LabelInstanceType = "beta.kubernetes.io/instance-type"
-
-	LabelOS   = "kubernetes.io/os"
-	LabelArch = "kubernetes.io/arch"
-	// The OS/Arch labels are promoted to GA in 1.14. kubelet applies both beta
-	// and GA labels to ensure backward compatibility.
-	// TODO: stop applying the beta OS/Arch labels in Kubernetes 1.17.
-	LegacyLabelOS   = "beta.kubernetes.io/os"
-	LegacyLabelArch = "beta.kubernetes.io/arch"
-
 	// GA versions of the legacy beta labels.
 	// TODO: update kubelet and controllers to set both beta and GA labels, then export these constants
 	labelZoneFailureDomainGA = "failure-domain.kubernetes.io/zone"
 	labelZoneRegionGA        = "failure-domain.kubernetes.io/region"
 	labelInstanceTypeGA      = "kubernetes.io/instance-type"
-
-	// LabelNamespaceSuffixKubelet is an allowed label namespace suffix kubelets can self-set ([*.]kubelet.kubernetes.io/*)
-	LabelNamespaceSuffixKubelet = "kubelet.kubernetes.io"
-	// LabelNamespaceSuffixNode is an allowed label namespace suffix kubelets can self-set ([*.]node.kubernetes.io/*)
-	LabelNamespaceSuffixNode = "node.kubernetes.io"
-
-	// LabelNamespaceNodeRestriction is a forbidden label namespace that kubelets may not self-set when the NodeRestriction admission plugin is enabled
-	LabelNamespaceNodeRestriction = "node-restriction.kubernetes.io"
 )
 
 // When the --failure-domains scheduler flag is not specified,
 // DefaultFailureDomains defines the set of label keys used when TopologyKey is empty in PreferredDuringScheduling anti-affinity.
-var DefaultFailureDomains string = LabelHostname + "," + LabelZoneFailureDomain + "," + LabelZoneRegion
+var DefaultFailureDomains string = v1.LabelHostname + "," + v1.LabelZoneFailureDomain + "," + v1.LabelZoneRegion
```

Comment:
Shouldn't `v1.LabelZoneRegion` looks like a brain-o.  Shouldn't it be `v1.LabelRegionFailureDomain`  to be parallel?

Reply:
I really really don't like that we have this codified as an API.  It's such a provider-specific detail.

Reply:
Ack fixing this